src/pomodoro.sh

```diff
@@ -389,6 +420,7 @@ function pomodoro_help()
     '  pomodoro (-s|--show-tags) - Show registered tags' \
     '  pomodoro (-t|--set-timer) <time>(h|m|s) (-g|--tag) <tag> - Set timer with tag' \
     '  pomodoro (-t|--set-timer) <time>(h|m|s) (-g|--tag) <tag> (-d|--description) <desc> - Set timer with tag and description' \
+    '  pomodoro (--repeat) - Repeat the last completed Pomodoro session' \
```

Comment:
I don't think that this is true. Even if the Pomodoro session isn't complete, you can repeat it in your implementation. When you update the PR to use the DB instead of the log file you created, the behavior will be the same, i.e., the session can be repeated before its completion.